packages/front-end/components/Features/Rule.tsx

```diff
@@ -60,6 +64,41 @@ export const Rule = forwardRef<HTMLDivElement, RuleProps>(
       permissions.check("manageFeatures", feature.project) &&
       permissions.check("createFeatureDrafts", feature.project);
 
+    const hasConditions = rule.condition && rule.condition !== "{}";
+
+    let upcomingScheduleRule: ScheduleRule | null = null;
+
+    const currentDate = new Date().valueOf();
+
+    const sortedScheduleRules = rule.scheduleRules.sort(
+      (a, b) =>
+        new Date(a.timestamp).valueOf() - new Date(b.timestamp).valueOf()
+    );
+
+    const nextRuleIndex = sortedScheduleRules.findIndex(
+      (rule) => currentDate < new Date(rule.timestamp).valueOf()
+    );
+
+    if (nextRuleIndex > -1) {
+      upcomingScheduleRule = sortedScheduleRules[nextRuleIndex];
+    }
+
+    let currentScheduleStatus;
+
+    if (!upcomingScheduleRule && rule.scheduleRules.length) {
+      currentScheduleStatus = rule.scheduleRules[rule.scheduleRules.length - 1]
+        .enableFeature
+        ? "enabled"
+        : "disabled";
+    }
+    if (!rule.scheduleRules || !rule.scheduleRules.length) {
+      currentScheduleStatus = "enabled";
+    }
+
+    const ruleDisabled =
```

Comment:
Can I update this to also include if `!rule.enabled`?

Reply:
No, it breaks in the event that a rule has been actually "Disabled" by the user AND the current scheduling rule has the feature "disabled". Ultimately, I just need to differentiate between enable/disable in the manual sense, and the "status" that is derived from the schedule rules.